documentation/tutorial.md

```diff
@@ -68,3 +71,416 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+      lastAppliedWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      lastDurableWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      lastHeartbeat: ISODate("2022-12-02T14:39:51.868Z"),
+      lastHeartbeatRecv: ISODate("2022-12-02T14:39:51.882Z"),
+      pingMs: Long("0"),
+      lastHeartbeatMessage: '',
+      syncSourceHost: '10.23.62.156:27017',
+      syncSourceId: 0,
+      infoMessage: '',
+      configVersion: 5,
+      configTerm: 1
+    },
+    {
+      _id: 2,
+      name: '10.23.62.243:27017',
+      health: 1,
+      state: 2,
+      stateStr: 'SECONDARY',
+      uptime: 300,
+      optime: { ts: Timestamp({ t: 1669991990, i: 1 }), t: Long("1") },
+      optimeDurable: { ts: Timestamp({ t: 1669991990, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-12-02T14:39:50.000Z"),
+      optimeDurableDate: ISODate("2022-12-02T14:39:50.000Z"),
+      lastAppliedWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      lastDurableWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      lastHeartbeat: ISODate("2022-12-02T14:39:51.861Z"),
+      lastHeartbeatRecv: ISODate("2022-12-02T14:39:52.372Z"),
+      pingMs: Long("0"),
+      lastHeartbeatMessage: '',
+      syncSourceHost: '10.23.62.55:27017',
+      syncSourceId: 1,
+      infoMessage: '',
+      configVersion: 5,
+      configTerm: 1
+    }
+  ],
+  ok: 1,
+  '$clusterTime': {
+    clusterTime: Timestamp({ t: 1669991990, i: 1 }),
+    signature: {
+      hash: Binary(Buffer.from("dbe96e73cf659617bb88b6ad11152551c0dd9c8d", "hex"), 0),
+      keyId: Long("7172510554420936709")
+    }
+  },
+  operationTime: Timestamp({ t: 1669991990, i: 1 })
+}
+```
+
+Now exit the shell by typing:
+```
+exit
+```
+
+
+### Remove replicas:
+To remove a replica type:
+```
+juju remove-unit mongodb/2
+```
+
+You’ll know that the replica was successfully removed when `watch -c juju status --color` reports:
+```
+Every 2.0s: juju status --color                                                                                                                                                ip-172-31-11-104: Fri Dec  2 14:44:25 2022
+
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  14:44:25Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      2  mongodb  dpe/edge   96  no       Replica set primary
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        10.23.62.156    27017/tcp  Replica set primary
+mongodb/1   active    idle   1        10.23.62.55     27017/tcp  Replica set secondary
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+1        started  10.23.62.55   juju-d35d30-1  focal       Running
+
+```
+
+As previously mentioned you can trust that Charmed MongoDB removed this replica correctly.
```

Comment:
Is there any check one could do here to make sure this has been done correctly? If we had the action to retrieve the URI, one could retrieve the new URI and make a check call. But if we don't have that info, I'm not sure. But, as a reader, when I read "you can trust" I'm thinking, "is there a way I can make sure or should I really take your word for"? :D

Reply:
good question we could repeat the steps from above and show the replica set configuration. (illustrating that the replica is removed in the config) what do you think of that?

 as for making an action to retrieve the URI - 
- creating a new action requires: creating specifications and approve them via the VP and PMs
- we could however update the `get-password` action such that it outputs the URI

thoughts?

Reply:
No, that's fine. By the way, the task at hand is about writing the tutorial. I would go down implementing actions for this. I'm just mentioning that I can see a use-case for this to be userful for the user. I would not repeat all the steps one more. Possibly we can say:

```suggestion
As previously mentioned you can trust that Charmed MongoDB removed this replica correctly. This can be checked however by verifying that the new URI (where the removed host has been excluded) works properly.
```